OrbitGl/ThreadTrack.cpp

```diff
@@ -126,7 +142,7 @@ void ThreadTrack::SetTimesliceText(const Timer& timer, double elapsed_us,
                                          time.c_str());
       text_box->SetText(text);
     } else {
-      ERROR("Unexpected case in ThreadTrack::SetTimesliceText");
+      ERROR("Unexpected case in TimerTrack::SetTimesliceText");
```

Comment:
Looks like this message should not have been changed, the method remains in ThreadTrack

Reply:
Thanks, I guess this was part of TimerTrack at some point during my refactor :)